deps/build.jl

```diff
@@ -1,5 +1,5 @@
-using Compat
-if is_windows()
+@static VERSION < v"0.7.0-DEV" || (const is_windows = Sys.iswindows)
+@static if is_windows()
```

Comment:
Why not just use `Compat` for this? Usually the standard is to use the latest Julia syntax so this is should be:
```julia
if Sys.iswindows()
```

Reply:
I did try that, with `using Compat`, but it kept complaining that iswindows was not defined.
I'm not sure what the issue is.